src/redecomp/partition/RCB.cpp

```diff
@@ -29,23 +29,29 @@ std::vector<EntityIndexByRank> RCB<NDIMS>::generatePartitioning(
   // Build a partitioning using recursive coordinate bisection
   auto problem_tree = BuildProblemTree( n_parts, coords_by_mesh, ghost_len );
 
+  // Ensures ranks with elements are evenly distributed throughout the partition
```

Comment:
Consider rewording? This reads like the ranks are evenly distributed, which I don't follow. I think we want the elements to be evenly distributed across the selected number of non-zero-element ranks.